FireBrick/UI/AddTechnologyViewController/AddTechnologyViewController.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  AddTechnologyViewController.swift
+//  FireBrick
+//
+//  Created by metoSimka on 27/05/2019.
+//  Copyright © 2019 metoSimka. All rights reserved.
+//
+
+import UIKit
+import SwiftEntryKit
+import Firebase
+import FirebaseFirestore
+
+class AddTechnologyViewController: UIViewController {
+    
+    var docRef: DocumentReference!
+    var db: Firestore?
```

Comment:
```suggestion
    private var firestore: Firestore?
```
needs to be removed probably